tool/internal/instrument/apply_func.go

```diff
@@ -297,6 +301,9 @@ func (ip *InstrumentPhase) parseFile(file string) (*dst.File, error) {
 		return nil, err
 	}
 	ip.target = root
+	// Every time we parse a file, we need to reset the trampoline jumps
+	// because they are associated with one certain file
+	ip.tjumps = make([]*TJump, 0)
```

Comment:
You could probably truncate it instead, as this would reduce re-allocations (nb -- off the top of my head I have a doubt whether this is `nil`-safe or not):


```suggestion
	ip.tjumps = ip.tjumps[:0]
```